src/stores.c

```diff
@@ -0,0 +1,47 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "stores.h"
+#include "items_types.h"
+#include "items.h"
+#include "api/fetch.h"
+#include "test_bargain.h"
+#include "test_functions.h"
+
+void stores_populate_store_items(store_s *stores, int store_count)
+{
+
+    for (int n = 0; n < store_count; n++)
+    {
+        // switch in case we add more chains
+        switch (stores[n].group)
+        {
+        case COOP:
+            // stores[n].items_count = test_get_items_one(&stores[n]);
+            fetch_get_coop_items(&stores[n]); // TODO: Implement when function get_coop_items has been implemented.
+            break;
+        case SALLING:
+            stores[n].items_count = test_get_items_one(&stores[n]);
+            // fetch_get_salling_items(&(stores[n])); // TODO: Implement when function get_salling_items has been implemented.
+            break;
+        default:
+            // TODO make able to print any type after got enum.
+            printf("Store[%d].chain did not match any known case. Got enum: %d \n", n, stores[n].chain);
```

Comment:
Here we're printing debug info about `store.chain`, but in the switch we're comparing to `store.group`